src/components/productDetails/list.jsx

```diff
@@ -17,19 +19,26 @@ const List = ({
   fetchRelatedDataAsync
   //avgRating,
 }) => {
+  // Set Local State For Outfits
   var [outfits, setOutfits] = useState([]);
   var [outfitsImages, setOutfitsImages] = useState([]);
 
+  // Set Local State For Conditionally Rendered Products
+  let initialShownIndices = listName === "Outfit" ? [0, 1, 2] : [0, 1, 2, 3];
+  var [shownIndices, setShownIndices] = useState(initialShownIndices);
+
+  //conditionally set outfits to Outfit data if this list represents outfits
   if (listName === "Outfit") {
     products = outfits;
     productsImages = outfitsImages;
   }
+
   //HANDLE FETCHING DATA ON AFTER FIRST RENDER
   useEffect(() => {
```

Comment:
Interesting use of use effect, didn't know about this behavior when there is no second argument